content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -81,93 +96,76 @@ $ istioctl install -f istio-cni.yaml -y
+{{< /tip >}}
+
+Note that if installing `istiod` with the Helm chart according to the [Install with Helm](/docs/setup/install/helm/#installation-steps) guide, you must install `istiod` with the following extra override value, in order to disable the privileged init container injection:
+
+{{< text bash >}}
+$ helm install istiod istio/istiod -n istio-system --set pilot.cni.enabled=true --wait
+{{< /text >}}
+
+#### Additional configuration for the `istio-cni` component
+
+In addition to the above basic configuration there are additional configuration flags that can be set:
 
-* `components.cni.namespace=kube-system` configures the namespace to install the CNI DaemonSet.
 * `values.cni.cniBinDir` and `values.cni.cniConfDir` configure the directory paths to install the plugin binary and create plugin configuration.
 * `values.cni.cniConfFileName` configures the name of the plugin configuration file.
 * `values.cni.chained` controls whether to configure the plugin as a chained CNI plugin.
 
+Normally, these do not need to be changed, but some platforms may use nonstandard paths. Please check the guidelines for your specific platform, if any, [here](/docs/ambient/install/platform-prerequisites)
+
 {{< tip >}}
 There is a time gap between a node becomes schedulable and the Istio CNI plugin becomes ready on that node.
 If an application pod starts up during this time, it is possible that traffic redirection is not properly set up and traffic would be able to bypass the Istio sidecar.
-This race condition is mitigated by a "detect and repair" method.
-Please take a look at [race condition & mitigation](#race-condition--mitigation) section to understand the implication of this mitigation.
-{{< /tip >}}
-
-### Installing with Helm
 
-Following the [Install with Helm](/docs/setup/install/helm/#installation-steps) installation steps, there are some additional Helm values that are required to be set. You can
-set these either through an override values file or via the command line when installing the `istiod` chart. These are:
-
-* `values.istio_cni.enabled` should be set to the same value as `values.cni.enabled`.
-* `values.istio_cni.chained` should be set to the same value as `values.cni.chained`.
-
-For example:
-
-{{< text bash >}}
-$ helm install istiod istio/istiod -n istio-system --set istio_cni.enabled=true --wait
-{{< /text >}}
-
-### Hosted Kubernetes settings
-
-The `istio-cni` plugin is expected to work with any hosted Kubernetes version using CNI plugins.
-The default installation configuration works with most platforms.
-Some platforms required special installation settings.
+This race condition is mitigated for the sidecar dataplane mode by a "detect and repair" method.
```

Comment:
Nit: I think we have settled on "sidecar data plane mode", or just "sidecar mode"

Reply:
I tried very hard to train my brain to think `data plane` everywhere and not `dataplane` but I missed this one :sweat_smile: